leetcode-helper.js

```diff
@@ -29,33 +29,51 @@
     GM_registerMenuCommand("复制 LeetCode 题目为 Markdown，并存入剪切板", copy);
     GM_registerMenuCommand("生成当前题目的题解模板，并存入剪切板", generateSolution);
 
+    // 按钮样式
+    var buttonStyle = {
+        width: "50px",
+        height: "25px",
+        fontSize: "14px",
+        color: "orange",
+        align: "center",
+        marginLeft: "10px",
+    };
+    
     // 添加复制按钮
     var copyBtn = document.createElement("button"); //创建一个 input 对象（提示框按钮）
     copyBtn.id = "copyBtn";
     copyBtn.innerHTML = '<i class="fa fa-copy"></i> 复制';
-    copyBtn.style.width = "50px";
-    copyBtn.style.height = "25px";
-    copyBtn.style.align = "center";
-    copyBtn.style.marginLeft = "10px";
+    Object.assign(copyBtn.style, buttonStyle);
     copyBtn.title = "复制题目为 Markdown 格式";
 
     // 添加生成题解按钮
 	var generateSolutionBtn = document.createElement("button"); // 创建一个input对象（提示框按钮）
 	generateSolutionBtn.id = "generateSolutionBtn";
 	generateSolutionBtn.innerHTML = '<i class="fa fa-edit"></i> 生成';
-	generateSolutionBtn.style.width = "50px";
-    generateSolutionBtn.style.height = "25px";
-	generateSolutionBtn.style.align = "center";
-    generateSolutionBtn.style.marginLeft = "10px";
+    Object.assign(generateSolutionBtn.style, buttonStyle);
     generateSolutionBtn.title = "生成 Markdown 格式题解";
 
-    window.onload = setTimeout(function () {
-        var x = document.getElementsByClassName("mr-2 text-label-1 dark:text-dark-label-1 text-lg font-medium")[0];
-        console.log("I was invoked...");
-        x.parentNode.appendChild(copyBtn);
-        x.parentNode.appendChild(generateSolutionBtn);
-    }, 1500);
-
+    window.onload = function () {
+        setTimeout(function () {
+            // 筛选出符合格式的href: '/problems/.*/'，使用两个selector串联，获取到的第一个为 /problems/，跳过
+            var x = $("a[href^='/problems/'][href$='/']")[1]
+            
+            // 等价于以下代码
```

Comment:
The commented code can be removed.

Reply:
> with a function outside | 在外面又套了一层function

I found this is just unnecessary (my wrong understanding). Only need to extend waiting time for my poor computer. Ignore please.

Btw, can it loop to check whether `parentNode` exists, and append button until loaded.

Reply:
I also reply you on tampermonkey.